triaging.rst

```diff
@@ -28,14 +28,21 @@ Responsibilities include:
 * PR/issue management
     - Renaming PRs
     - Reviewing PRs
-    - Closing PRs and issues
     - Assisting contributors
     - Notifying appropriate core developers
 * Applying appropriate labels to PRs/Issues
     - Skip news
     - Skip issue
     - Good first issue
     - Other categorizations
+    
+ .. note::
+ 
+   Although triagers have the permission to close PRs, a core
+   developer must take the final action in doing so. A triager should
```

Comment:
Maybe explain the rationale here? Something along the lines of, "This is because a PR constitutes a significant effort by the author. Closing pull requests without proper rationale and careful consideration can dissuade people from contributing."

Reply:
@ammaraskar 
> Maybe explain the rationale here?

That's a good idea. I'll think about a good way phrase it, based on the example you provided.